content/en/agent/kubernetes/apm.md

```diff
@@ -34,37 +34,29 @@ You can configure the Agent to intake traces by using either `IP:Port`, Unix Dom
-      apm:
-        # datadog.apm.portEnabled -- Enable APM over TCP communication (port 8126 by default)
-        ## ref: https://docs.datadoghq.com/agent/kubernetes/apm/
-        portEnabled: true
-    ```
 
-Then, upgrade your Datadog Helm chart using the following command: `helm upgrade -f values.yaml <RELEASE NAME> datadog/datadog`. If you did not set your operating system in `values.yaml`, add `--set targetSystem=linux` or `--set targetSystem=windows` to this command.
+The default configuration creates a directory on the host and mounts it within the Agent. The Agent then creates and listens on a socket file `/var/run/datadog/apm.socket`. The application pods can then similarly mount this volume and write to this same socket. You can modify the path and socket with the `datadog.apm.hostSocketPath` and `datadog.apm.socketPath` configuration values. 
 
-[1]: /agent/kubernetes/?tab=helm
-{{% /tab %}}
-{{% tab "Helm (UDS)" %}}
+This feature can be disabled with `datadog.apm.socketEnabled`.
+
+#### Optional - Configure the Datadog Agent to accept traces over TCP
+
+Datadog agent can also be configured to receive traces over TCP. To enable this feature:
 
-- If you haven't already, [install][1] the Helm chart.
 - Update your `values.yaml` file with the following APM configuration:
     ```yaml
     datadog:
       ## Enable apm agent and provide custom configs
       apm:
-        # datadog.apm.socketEnabled -- Enable APM over Socket (Unix Socket or windows named pipe)
+        # datadog.apm.portEnabled -- Enable APM over TCP communication (port 8126 by default)
         ## ref: https://docs.datadoghq.com/agent/kubernetes/apm/
-        socketEnabled: true
+        portEnabled: true
     ```
 
-This configuration creates a directory on the host and mounts it within the Agent. The Agent then creates and listens on a socket file `/var/run/datadog/apm.socket`. The application pods can then similarly mount this volume and write to this same socket. You can modify the path and socket with the `datadog.apm.hostSocketPath` and `datadog.apm.socketPath` configuration values.
-
 Then, upgrade your Datadog Helm chart using the following command: `helm upgrade -f values.yaml <RELEASE NAME> datadog/datadog`. If you did not set your operating system in `values.yaml`, add `--set targetSystem=linux` or `--set targetSystem=windows` to this command.
 
+**Warning**: The `datadog.apm.portEnabled` parameter opens a port on your host. Make sure your firewall only allows access from your applications or trusted sources. If your network plugin doesn’t support `hostPorts`, so add `hostNetwork: true` in your Agent pod specifications. This shares the network namespace of your host with the Datadog Agent. It also means that all ports opened on the container are opened on the host. If a port is used both on the host and in your container, they conflict (since they share the same network namespace) and the pod does not start. Some Kubernetes installations do not allow this.
```

Comment:
```suggestion
**Warning**: The `datadog.apm.portEnabled` parameter opens a port on your host. Make sure your firewall only allows access from your applications or trusted sources. If your network plugin doesn’t support `hostPorts`, add `hostNetwork: true` in your Agent pod specifications. This shares the network namespace of your host with the Datadog Agent. This also means that all ports opened on the container are opened on the host. If a port is used both on the host and in your container, they conflict (since they share the same network namespace) and the pod does not start. Some Kubernetes installations do not allow this.
```